deployment/cdk/opensearch-service-migration/options.md

```diff
@@ -13,6 +13,7 @@ These tables list all CDK context configuration values a user can specify for th
 | captureProxyESServiceEnabled         | boolean | true                                                                   | Enable deploying the given service, via a new CloudFormation stack                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
 | captureProxyESExtraArgs              | string  | `"--suppressCaptureForHeaderMatch user-agent .*elastic-java/7.17.0.*"` | Extra arguments to provide to the Capture Proxy command. This includes available arguments specified by the [Capture Proxy](../../../TrafficCapture/trafficCaptureProxyServer/src/main/java/org/opensearch/migrations/trafficcapture/proxyserver/CaptureProxy.java).                                                                                                                                                                                                                                                                                                                                                               |
 | migrationConsoleServiceEnabled       | boolean | true                                                                   | Enable deploying the given service, via a new CloudFormation stack                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
+| migrationConsoleDjangoAPIEnabled     | boolean | true                                                                   | **Experimental** flag for enabling a basic Django API layer on the Migration Console for controlling migration actions                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
```

Comment:
Lets remove the technology name from everything that doesn't need it.  While its Django today, we don't need to expose it outside its actual implementation, no?

```suggestion
| migrationAPIEnabled     | boolean | true                                                                   | **Experimental** flag for enabling a API on the Migration Console for controlling migration actions                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
```